app/api/access_codes.py

```diff
@@ -1,13 +1,14 @@
 from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
 from flask_rest_jsonapi.exceptions import ObjectNotFound
 from sqlalchemy.orm.exc import NoResultFound
-
 from app.api.bootstrap import api
 from app.api.helpers.db import safe_query
 from app.api.helpers.exceptions import (
-    ConflictException,
-    ForbiddenException,
-    UnprocessableEntity,
+    ConflictException
```

Comment:
Why is this left?

Reply:
I didn't find conflictException in other file(error.py).

Reply:
Then move it there

Reply:
so I should remove all exceptions from exception.py and move to error.py.there are only two left ConflictException and MethodNotAllowed?

Reply:
@iamareebjamal Shall I move all exception from exception.py to error.py ???

Reply:
Yup

Reply:
Yup